src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -22,30 +22,114 @@
 *
 * Author: Ken Zangelin
 */
-
+#include <sys/time.h>
```

Comment:
Not sure but probably after the removal of the time-related function, this include is no longer needed.